db/migrate/20220520210546_create_posts.rb

```diff
@@ -0,0 +1,9 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :name
+
```

Comment:
The posts table should have a column name `title` of `string` data type, `text` of `text` data type, `likes_counter` of `integer` data type, and `comments_counter` of `integer` data type.